values.yaml

```diff
@@ -4,6 +4,7 @@
 
 templates:
   bilateralSettlement: true
+  multilateralSettlement: true
```

Comment:
I'd suggest adding new keys in alphabetic order

Reply:
Done